packages/@atjson/renderer-commonmark/src/index.ts

```diff
@@ -232,12 +263,26 @@ function getNumberOfRequiredBackticks(text: string) {
     }
   }
 
-  return counts.sort().reduce((result, count) => {
-    if (count === result) {
-      return result + 1;
+  let total = 1;
+  for (let count of counts) {
+    if (count === total) {
+      total += 1;
     }
-    return result;
-  }, 1);
+  }
+
+  return total;
+}
+
+function isHTML(a: { type: string }) {
```

Comment:
This _looks_ like a helper for doing type narrowing on HTML; you could add a `a is HTML` to the end of this declaration